department-of-veterans-affairs/caseflow
# APPEALS-56527: MST and PACT values being set to null when unidentified issue added

Resolves [APPEALS-56527](https://jira.devops.va.gov/browse/APPEALS-56527)

# Description
Added in logic to add **false** MST/PACT statuses for unidentified issues.

## Acceptance Criteria

## Testing Plan

1. Log in as BVADWISE
2. Navigate to Intake
3. Select **10182** from the dropdown
4. enter the veteran file number 981775045
5. Fill out the form
6. On the Add Issues page, click **add issue**
7. Click **none of these match**.
8. Click **none of these match again**.
9. Fill out an issue description and add the issue. 
10. Expect the issue to appear in red on the issues page. Also expect the Special Issue status to be **none**.
11. Click **establish appeal**
12. Validate the appeal can be established. 
13. Click the **correct issues link**.
14. Repeat steps 6-10 on the Add Issues page. 
15. On the originally added issue, click the **actions** dropdown and select **edit issue**.
16. Select MST from the dropdown.
17. Validate that the MST status on the issue is updated to show MST.
18. Click **save**. Validate that the user is directed to the Case Details page with a green success banner showing the issue was added and the other issue was updated. Validate the IssueUpdateTask is in the timeline. 


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
